website/pages/docs/acl/auth-methods/jwt.mdx

```diff
@@ -0,0 +1,173 @@
+---
+layout: docs
+page_title: JWT Auth Method
+sidebar_title: JWT
+description: >-
+  The JWT auth method can be used to authenticate with Consul by providing a
+  JWT directly. The JWT is cryptographically verified using locally-provided
+  keys, or, if configured, an OIDC Discovery service can be used to fetch the
+  appropriate keys.
+---
+
+# JWT Auth Method
+
+-> **1.8.0+:** This feature is available in Consul versions 1.8.0 and newer.
+
+The `jwt` auth method can be used to authenticate with Consul by providing a
+[JWT](https://en.wikipedia.org/wiki/JSON_Web_Token) directly.  The JWT is
+cryptographically verified using locally-provided keys, or, if configured, an
+OIDC Discovery service can be used to fetch the appropriate keys.
+
+This page assumes general knowledge of JWTs and the concepts described in the
+main [auth method documentation](/docs/acl/auth-methods).
+
+Both the [`jwt`](/docs/acl/auth-methods/jwt) and the
+[`oidc`](/docs/acl/auth-methods/oidc) auth method types allow additional
+processing of the claims data in the JWT.
```

Comment:
Is it worth noting the difference between OIDC and JWT here? Or linking to a section describing the difference elsewhere?